docs/installation/add-to-a-website.md

```diff
@@ -74,26 +79,279 @@ In order to do this, you'll need to build your website with a JavaScript bundler
 
+```sh
+$ npm install history@5 react-router@6 react-router-dom@6
 ```
-$ yarn add react-router@6 react-router-dom@6
+
+## Create React App
+
+Follow the instructions in the [React documentation to set up a new project with Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), then follow [the instructions above](/#install-with-a-package-manager)
+to install React Router in your project.
+
+Once your project is set up and React Router is installed as a dependency, open
+the `src/index.js` in your text editor. Import `BrowserRouter` from
+`react-router-dom` near the top of your file.
+
+```diff
+import React from 'react';
+import ReactDOM from 'react-dom';
++ import { BrowserRouter as Router } from "react-router-dom";
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+```
+
+To make sure your app has the necessary context to handle state and logic
+controlled by React Router, you need to wrap the entire app inside the
+`BrowserRouter` component. Where `ReactDOM.render` is called to
+load your app, update the first argument so that `<App />` is nested
+accordiongly.
+
+```diff
+- ReactDOM.render(<App />, document.getElementById('root'));
++ ReactDOM.render(
++   <Router>
++     <App />
++   </Router>,
++   document.getElementById("root")
++ );
+```
+
+Now you can use React Router anywhere in your app! For a simple example, open
+`src/App.js` and give your app some routes:
+
+```diff
+import React from 'react';
++ import { Routes, Route, Link } from 'react-router-dom';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <h1>Welcome to React Router!</h1>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
++     <Routes>
++       <Route path="/" element={<Home />} />
++       <Route path="about" element={<About />} />
++     </Routes>
+    </div>
+  );
+}
+```
+
+Now create your route components:
+
+```js
+// App.js
+function Home() {
+  return (
+    <React.Fragment>
+      <main>
+        <h2>Welcome to the homepage!</h2>
+        <p>You can do this, I believe in you.</p>
+      </main>
+      <nav>
+        <Link to="/about">About</Link>
+      </nav>
+    </React.Fragment>
+  );
+}
+
+function About() {
+  return (
+    <React.Fragment>
+      <main>
+        <h2>Who are we?</h2>
+        <p>That feels like an existential question, don't you think?</p>
+      </main>
+      <nav>
+        <Link to="/">Home</Link>
+      </nav>
+    </React.Fragment>
+  );
+}
+```
+
+Now start your app by running `npm start`, and you should see the `Home` route
+when your app starts runinng. Click the `About` link to see your `About` route,
+and voila! You successfully set up React Router using Create React App! 🥳
```

Comment:
The CRA docs have a great guide to deploying a CRA + React Router app that is probably worth linking to here. See https://create-react-app.dev/docs/deployment#serving-apps-with-client-side-routing